### ISSUE TITLE
Add a bigger margin to the bottom of report view

### ISSUE THREAD

**Issue by maritoggl** [toggl/mobileapp#6111]
Currently, when scrolling down in the report view, the workspace selection button covers the name of a project in the list. We should have some sort of a margin there so that none of the project names would be hidden behind the workspace selection button (similarily to what we have for the new time entry button in the time log view)

![IMG_12D15557C971-1](https://user-images.githubusercontent.com/35216651/66643418-55455e80-ec27-11e9-8ad6-f0feeb48c958.jpeg)

**Comment by simonrozsival** [comment]
I fixed this in #6068

**Comment by simonrozsival** [comment]
It will be fixed on `develop`, once #6053 is merged into develop.

**Comment by maritoggl** [comment]
Alright then, closing :)
